The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/6386533675/job/17333350714) [(plain)](https://github.com/rust-lang/rust/commit/f96cfb533a5cb84277dd894854fbac44628626cf/checks/17333350714/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
Prepare all required actions
Getting action download info
Download action repository 'actions/checkout@v4' (SHA:8ade135a41bc03ea155e62e844d188df1ea18608)
Download action repository 'actions/upload-artifact@v3' (SHA:a8a3f3ad30e3422c9c7b888a15615d19a852ae32)
Complete job name: PR - mingw-check-tidy
git config --global core.autocrlf false
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
---
GITHUB_ACTION=__run_7
GITHUB_ACTIONS=true
GITHUB_ACTION_REF=
GITHUB_ACTION_REPOSITORY=
GITHUB_ACTOR=Colonial-Dev
GITHUB_API_URL=https://api.github.com
GITHUB_BASE_REF=master
GITHUB_ENV=/home/runner/work/_temp/_runner_file_commands/set_env_4541aee1-00a2-476f-a00c-12d55b99b2aa
GITHUB_EVENT_NAME=pull_request
---
GITHUB_SERVER_URL=https://github.com
GITHUB_SHA=d700abe2614fe064fc436a1a9b81bd6b00296326
GITHUB_STATE=/home/runner/work/_temp/_runner_file_commands/save_state_4541aee1-00a2-476f-a00c-12d55b99b2aa
GITHUB_STEP_SUMMARY=/home/runner/work/_temp/_runner_file_commands/step_summary_4541aee1-00a2-476f-a00c-12d55b99b2aa
GITHUB_TRIGGERING_ACTOR=Colonial-Dev
GITHUB_WORKFLOW_REF=rust-lang/rust/.github/workflows/ci.yml@refs/pull/116363/merge
GITHUB_WORKFLOW_SHA=d700abe2614fe064fc436a1a9b81bd6b00296326
GITHUB_WORKSPACE=/home/runner/work/rust/rust
GOROOT_1_19_X64=/opt/hostedtoolcache/go/1.19.13/x64
---
Removing intermediate container b967b2bd7a77
 ---> e820152f5772
Step 6/10 : COPY host-x86_64/mingw-check/reuse-requirements.txt /tmp/
 ---> 6fa7e55e57b3
Step 7/10 : RUN pip3 install --no-deps --no-cache-dir --require-hashes -r /tmp/reuse-requirements.txt     && pip3 install virtualenv
Collecting binaryornot==0.4.4
  Downloading binaryornot-0.4.4-py2.py3-none-any.whl (9.0 kB)
Collecting boolean-py==4.0
  Downloading boolean.py-4.0-py3-none-any.whl (25 kB)
---
Building wheels for collected packages: reuse
  Building wheel for reuse (pyproject.toml): started
  Building wheel for reuse (pyproject.toml): finished with status 'done'
  Created wheel for reuse: filename=reuse-1.1.0-cp310-cp310-manylinux_2_35_x86_64.whl size=180117 sha256=2196c9034bf565528bbb1ee6dad4f753eb813f58822363e6b768f09c73e4d4ff
  Stored in directory: /tmp/pip-ephem-wheel-cache-0nxy0_b9/wheels/c2/3c/b9/1120c2ab4bd82694f7e6f0537dc5b9a085c13e2c69a8d0c76d
Installing collected packages: boolean-py, binaryornot, setuptools, reuse, python-debian, markupsafe, license-expression, jinja2, chardet
  Attempting uninstall: setuptools
    Found existing installation: setuptools 59.6.0
    Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
    Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
    Can't uninstall 'setuptools'. No files were found to uninstall.
Successfully installed binaryornot-0.4.4 boolean-py-4.0 chardet-5.1.0 jinja2-3.1.2 license-expression-30.0.0 markupsafe-2.1.1 python-debian-0.1.49 reuse-1.1.0 setuptools-66.0.0
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
Collecting virtualenv
  Downloading virtualenv-20.24.5-py3-none-any.whl (3.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.7/3.7 MB 24.5 MB/s eta 0:00:00
Collecting distlib<1,>=0.3.7
  Downloading distlib-0.3.7-py2.py3-none-any.whl (468 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 49.3 MB/s eta 0:00:00
Collecting platformdirs<4,>=3.9.1
  Downloading platformdirs-3.11.0-py3-none-any.whl (17 kB)
Collecting filelock<4,>=3.12.2
  Downloading filelock-3.12.4-py3-none-any.whl (11 kB)
Installing collected packages: distlib, platformdirs, filelock, virtualenv
Successfully installed distlib-0.3.7 filelock-3.12.4 platformdirs-3.11.0 virtualenv-20.24.5
Removing intermediate container fdf8c44c0899
 ---> 1793dd42dcbd
Step 8/10 : COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
 ---> e6a5aa18d5bc
 ---> e6a5aa18d5bc
Step 9/10 : COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/
 ---> 09390ea0b118
Step 10/10 : ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
Removing intermediate container 3fc6567bc08a
 ---> 5ed6190c5a3f
Successfully built 5ed6190c5a3f
Successfully tagged rust-ci:latest
Successfully tagged rust-ci:latest
##[endgroup]
Built container sha256:5ed6190c5a3f9eea08fc8297cf6cefa54a119dfd914d98b045c1e527224ddd95
Uploading finished image sha256:5ed6190c5a3f9eea08fc8297cf6cefa54a119dfd914d98b045c1e527224ddd95 to https://ci-caches.rust-lang.org/docker/8849b25aebb63c7041ab10114da59fac9c6c89ff409673e53f6251b7e63c69daeaca7298d30885d05004ab27b231421908523f297222d07a53450f37e4691d72
IMAGE          CREATED              CREATED BY                                      SIZE      COMMENT
5ed6190c5a3f   1 second ago         /bin/sh -c #(nop)  ENV SCRIPT=TIDY_PRINT_DIF…   0B        
e6a5aa18d5bc   2 seconds ago        /bin/sh -c #(nop) COPY file:078ea1d11e7b7cda…   367B      
1793dd42dcbd   4 seconds ago        |1 DEBIAN_FRONTEND=noninteractive /bin/sh -c…   23.9MB    
6fa7e55e57b3   12 seconds ago       /bin/sh -c #(nop) COPY file:ac591dd6bc5afa66…   5.33kB    
e820152f5772   13 seconds ago       |1 DEBIAN_FRONTEND=noninteractive /bin/sh -c…   23.1MB    
---
<missing>      6 weeks ago          /bin/sh -c #(nop)  LABEL org.opencontainers.…   0B        
<missing>      6 weeks ago          /bin/sh -c #(nop)  ARG LAUNCHPAD_BUILD_ARCH     0B        
<missing>      6 weeks ago          /bin/sh -c #(nop)  ARG RELEASE                  0B        

<botocore.awsrequest.AWSRequest object at 0x7f634d7cb350>
gzip: stdout: Broken pipe
xargs: docker: terminated by signal 13
https://ci-caches.rust-lang.org/docker/8849b25aebb63c7041ab10114da59fac9c6c89ff409673e53f6251b7e63c69daeaca7298d30885d05004ab27b231421908523f297222d07a53450f37e4691d72
sha256:5ed6190c5a3f9eea08fc8297cf6cefa54a119dfd914d98b045c1e527224ddd95
---
DirectMap4k:      270272 kB
DirectMap2M:     7069696 kB
DirectMap1G:    11534336 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
    Finished dev [unoptimized] target(s) in 0.03s
##[endgroup]
downloading https://ci-artifacts.rust-lang.org/rustc-builds-alt/5333b878c8bc1c4267a67ea3682663629e47541a/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz
extracting /checkout/obj/build/cache/llvm-5333b878c8bc1c4267a67ea3682663629e47541a-true/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz to /checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm
---
   Compiling tidy v0.1.0 (/checkout/src/tools/tidy)
    Finished release [optimized] target(s) in 26.01s
##[endgroup]
fmt check
##[error]Diff in /checkout/library/core/src/macros/mod.rs at line 719:
 /// The difference between `unimplemented!` and [`todo!`] is that while `todo!`
 /// conveys an intent of implementing the functionality later and the message is "not yet
 /// implemented", `unimplemented!` makes no such claims. Its message is "not implemented".
-/// 
+///
 /// Also, some IDEs will mark `todo!`s.
Running `"/checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt/bin/rustfmt" "--config-path" "/checkout" "--edition" "2021" "--unstable-features" "--skip-children" "--check" "/checkout/library/core/src/result.rs" "/checkout/library/core/src/error.rs" "/checkout/library/core/src/escape.rs" "/checkout/library/core/src/intrinsics/mir.rs" "/checkout/library/core/src/any.rs" "/checkout/library/core/src/macros/mod.rs" "/checkout/library/core/src/ops/range.rs" "/checkout/library/core/src/ascii.rs"` failed.
If you're running `tidy`, try again with `--bless`. Or, if you just want to format code, run `./x.py fmt` instead.
 ///
 /// # Panics
 /// # Panics
##[error]Diff in /checkout/library/core/src/macros/mod.rs at line 806:
 /// The difference between [`unimplemented!`] and `todo!` is that while `todo!` conveys
 /// an intent of implementing the functionality later and the message is "not yet
 /// implemented", `unimplemented!` makes no such claims. Its message is "not implemented".
-/// 
+///
 /// Also, some IDEs will mark `todo!`s.
 /// # Panics
 /// # Panics
##[error]Diff in /checkout/library/core/src/macros/mod.rs at line 815:
 /// fixed, specific message.
 ///
 /// Like `panic!`, this macro has a second form for displaying custom values.
-/// 
 /// # Examples
 ///
 ///
 /// Here's an example of some in-progress code. We have a trait `Foo`:
##[error]Diff in /checkout/library/core/src/macros/mod.rs at line 849:
 ///         // Let's not worry about implementing baz() for now
 ///         todo!();
-/// 
+///
+///
 ///     fn qux(&self) -> Result<u64, ()> {
 ///         // We can add a message to todo! to display our omission.
  local time: Mon Oct  2 23:21:03 UTC 2023
  network time: Mon, 02 Oct 2023 23:21:03 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/library/core/src/macros/mod.rs
+++ b/library/core/src/macros/mod.rs
@@ -719,7 +719,7 @@ macro_rules! unreachable {
 /// The difference between `unimplemented!` and [`todo!`] is that while `todo!`
 /// conveys an intent of implementing the functionality later and the message is "not yet
 /// implemented", `unimplemented!` makes no such claims. Its message is "not implemented".
-/// 
+///
 /// Also, some IDEs will mark `todo!`s.
 ///
 /// # Panics
@@ -806,7 +806,7 @@ macro_rules! unimplemented {
 /// The difference between [`unimplemented!`] and `todo!` is that while `todo!` conveys
 /// an intent of implementing the functionality later and the message is "not yet
 /// implemented", `unimplemented!` makes no such claims. Its message is "not implemented".
-/// 
+///
 /// Also, some IDEs will mark `todo!`s.
 ///
 /// # Panics
@@ -815,7 +815,7 @@ macro_rules! unimplemented {
 /// fixed, specific message.
 ///
 /// Like `panic!`, this macro has a second form for displaying custom values.
-/// 
+///
 /// # Examples
 ///
 /// Here's an example of some in-progress code. We have a trait `Foo`:
@@ -849,7 +849,7 @@ macro_rules! unimplemented {
 ///         // Let's not worry about implementing baz() for now
 ///         todo!();
 ///     }
-/// 
+///
 ///     fn qux(&self) -> Result<u64, ()> {
 ///         // We can add a message to todo! to display our omission.
 ///         // This will display:

